app/src/JokeFetch.tsx

```diff
@@ -108,36 +110,40 @@ const DURATION = "6s";
 
 const jump = keyframes`
   33% {
-    text-shadow: 0 60px #f37121, 0 150px #f2aaaa;
+    text-shadow: 0 60px #f37121, 0 100px #f2aaaa;
  }
   50% {
     transform: translate(0, 0) rotate(-4deg);
     text-shadow: 0 0px #8fc0a9, 0 0px #84a9ac;
  }
   66.67% {
-    text-shadow: 0 -60px #d54062, 0 -150px #8fc0a9;
+    text-shadow: 0 -60px #d54062, 0 -100px #8fc0a9;
  }
 `;
 
+const Container = styled.div`
```

Comment:
Great use of flex styling here, well done, I love to see styling implemented, shows how creative you are 🙌 🤩